CHANGELOG.md

```diff
@@ -14,6 +14,74 @@
 * [CHANGE] Update Alpine image version to 3.19 [#3289](https://github.com/grafana/tempo/pull/3289) (@zalegrala)
 * [CHANGE] Introduce localblocks process config option to select only server spans 3303https://github.com/grafana/tempo/pull/3303 (@zalegrala)
 * [CHANGE] Localblocks processor honor tenant max trace size limit [3305](https://github.com/grafana/tempo/pull/3305) (@mdisibio)
+* [CHANGE] Continue evaluating spanmetrics filter policies even if not matched in one policy [#3308](https://github.com/grafana/tempo/pull/3308) (@zalegrala)
+  **BREAKING CHANGE** Exclude policies are no longer global
```

Comment:
This is an excellent write-up of the breaking change. Thank you. We can add this to the Upgrade page when we do the next release.